Podfile

```diff
@@ -2,27 +2,22 @@ platform :ios, '9.0'
 use_frameworks!
 
 def shared_pods
-    # This uses a local build of the Maps SDK if you are installing from within mapbox-gl-native
-    # Build the framework first with `make iframework` to use this option.
-    if Dir.pwd.include?('mapbox-gl-native')
```

Comment:
Do we still need this?

Reply:
Nope - it was implemented for something we're no longer going to do in GL-native. I'd rather take it out.

Reply:
Hrm, can you link me to that discussion, @captainbarbosa?